### ISSUE TITLE
[Bug] Chia syncing freezes at 5,352,275

### ISSUE THREAD

**Issue by LarsThePenguin** [Chia-Network/chia-blockchain#18082]
### What happened?

After downloading the (Blockchain Database BitTorrent Checkpoints)[https://www.chia.net/2023/03/19/introducing-chia-blockchain-database-bittorent-checkpoints/] and moving it to where my config.yaml says the database file is (an external SanDisk SSD), I left it on overnight, and when I woke up, It has been stuck at 5,352,275.
Note: All of my peers are ahead of me, and I'm using the node.chia.net:8444 introducer node.

### Version

2.1.3

### What platform are you using?

Windows

### What ui mode are you using?

GUI

### Relevant log output

```shell
https://drive.google.com/file/d/1eUL_dtLW9Ly3IWf-x7o0PSZD-sVh58Up/view?usp=sharing
(Google drive link due to log so large, it broke Github)
```

**Comment by LarsThePenguin** [comment]
I tried again, got to 5,400,000 (Only 4000 behind ): ), then it returned the exact same error

**Comment by wjblanke** [comment]
I'd recommend you update to Chia 2.3 just so everyone is on the same page. We think your issue is network related, there are certainly loads of log messages with websocket failures. Are you running a software firewall or network appliance that could be filtering things? Can you turn these off temporarily to test? Have you ever had syncing successful on this machine? Another test would be to delete your config.yaml file (make a backup) and have chia recreate it. There could be some misconfigured setting there.

**Comment by LarsThePenguin** [comment]
I have upgraded to Chia 2.3, I have no firewalls (None on my desktop, and none on my router), I believe I have synced before (before changing config.yaml), I am going to replace config.yaml but still change the database location (I don't have enough storage on my main drive).

**Comment by LarsThePenguin** [comment]
Didn't work

**Comment by wjblanke** [comment]
Can u set your log to INFO so we can see what is happening in the logs. WARNING doesn't show much. 

You could also try running without the GUI
https://docs.chia.net/installation/?_highlight=install#cli

chia start node

**Comment by emlowe** [comment]
Closing - you may comment and reopen with additional details as needed

**Comment by LarsThePenguin** [comment]
I downloaded the 7/1/24 checkpoint from Chia, and got the same error so yeah.
I retried with the log level at INFO, and here's the result (I don't speak wizard magic, so I don't know what it means):
```2024-08-04T18:17:22.467 wallet wallet                     : INFO     Starting service wallet ...
2024-08-04T18:17:22.468 wallet wallet                     : INFO     chia-blockchain version: 2.4.2
2024-08-04T18:17:22.469 wallet wallet_server              : INFO     Service capabilities: [(1, '1'), (2, '1'), (3, '1')]
2024-08-04T18:17:22.477 wallet chia.wallet.wallet_node    : INFO     Selected multiprocessing start method: default
2024-08-04T18:17:22.532 harvester chia.harvester.harvester: INFO     Using plots_refresh_parameter: {'batch_size': 300, 'batch_sleep_milliseconds': 1, 'interval_seconds': 120, 'retry_invalid_seconds': 1200}
2024-08-04T18:17:22.533 harvester harvester               : INFO     Starting service harvester ...
2024-08-04T18:17:22.534 harvester harvester               : INFO     chia-blockchain version: 2.4.2
2024-08-04T18:17:22.534 harvester harvester_server        : INFO     Service capabilities: [(1, '1'), (2, '1'), (3, '1')]
2024-08-04T18:17:22.541 harvester harvester               : INFO     Started harvester service on network_id: mainnet at port None
2024-08-04T18:17:22.547 harvester harvester               : INFO     Add resolved PeerInfo(_ip=IPv4Address('127.0.0.1'), _port=8447)
2024-08-04T18:17:22.557 harvester harvester_server        : INFO     Connected with farmer PeerInfo(_ip=IPv4Address('127.0.0.1'), _port=8447)
2024-08-04T18:17:22.560 harvester chia.plotting.cache     : INFO     Loaded 10 bytes of cached data
2024-08-04T18:17:22.561 harvester chia.plotting.cache     : INFO     Parsed 0 cache entries in 0.00s
2024-08-04T18:17:22.565 wallet chia.server.address_manager_store: INFO     Creating new address_manager
2024-08-04T18:17:22.658 harvester chia.harvester.harvester: INFO     _plot_refresh_callback: event started, loaded 0, removed 0, processed 0, remaining 0, duration: 0.00 seconds, total plots: 0
2024-08-04T18:17:22.659 harvester chia.harvester.harvester: INFO     _plot_refresh_callback: event done, loaded 0, removed 0, processed 0, remaining 0, duration: 0.00 seconds, total plots: 0
2024-08-04T18:17:23.027 wallet chia.wallet.wallet_node    : INFO     Wallet is logged in using key with fingerprint: 523351724
2024-08-04T18:17:23.032 wallet chia.wallet.wallet_node    : INFO     Updated last used fingerprint: 523351724
2024-08-04T18:17:23.041 wallet wallet                     : INFO     Started wallet service on network_id: mainnet at port None
2024-08-04T18:17:23.047 wallet wallet                     : INFO     Add resolved PeerInfo(_ip=IPv4Address('127.0.0.1'), _port=8444)
2024-08-04T18:17:23.056 wallet wallet_server              : INFO     Connected with full_node PeerInfo(_ip=IPv4Address('127.0.0.1'), _port=8444)
2024-08-04T18:17:26.915 wallet chia.wallet.wallet_node    : INFO     Received 32 peers from DNS seeder, using rdtype = A.
2024-08-04T18:17:29.153 wallet chia.wallet.wallet_node    : INFO     Received 32 peers from DNS seeder, using rdtype = AAAA.
2024-08-04T18:17:29.784 wallet chia.wallet.wallet_node    : INFO     Connected peer PeerInfo(_ip=IPv4Address('136.50.22.38'), _port=8444) is trusted: False
2024-08-04T18:17:29.786 wallet wallet_server              : INFO     Connected with full_node PeerInfo(_ip=IPv4Address('136.50.22.38'), _port=8444)
2024-08-04T18:17:29.965 wallet chia.wallet.wallet_node_api: INFO     Wallet received 1000 peers.
2024-08-04T18:17:31.583 wallet chia.wallet.wallet_node    : INFO     Connected peer PeerInfo(_ip=IPv4Address('114.85.88.55'), _port=8444) is trusted: False
2024-08-04T18:17:31.585 wallet wallet_server              : INFO     Connected with full_node PeerInfo(_ip=IPv4Address('114.85.88.55'), _port=8444)
2024-08-04T18:17:32.028 wallet chia.wallet.wallet_node_api: INFO     Wallet received 1000 peers.
2024-08-04T18:17:34.327 wallet chia.full_node.weight_proof: INFO     validating 14950 sub epochs
2024-08-04T18:17:34.371 wallet chia.full_node.weight_proof: INFO     validate weight proof peak height 5740911
2024-08-04T18:17:39.765 full_node chia.full_node.full_node: ERROR    sync from fork point failed: DatabaseError: database disk image is malformed
Traceback (most recent call last):
  File "chia\util\log_exceptions.py", line 20, in log_exceptions
  File "chia\full_node\full_node.py", line 1172, in sync_from_fork_point
  File "chia\full_node\full_node.py", line 1138, in validate_block_batches
  File "chia\full_node\full_node.py", line 1306, in add_block_batch
  File "chia\consensus\blockchain.py", line 427, in add_block
  File "chia\consensus\block_body_validation.py", line 415, in validate_block_body
  File "chia\full_node\coin_store.py", line 171, in get_coin_records
  File "aiosqlite\cursor.py", line 76, in fetchall
  File "aiosqlite\cursor.py", line 40, in _execute
  File "aiosqlite\core.py", line 132, in _execute
  File "aiosqlite\core.py", line 115, in run
sqlite3.DatabaseError: database disk image is malformed

2024-08-04T18:17:52.807 full_node asyncio                 : ERROR    Task was destroyed but it is pending!
task: <Task pending name='Task-2047' coro=<FullNode.sync_from_fork_point.<locals>.fetch_block_batches() running at chia\full_node\full_node.py:1100> wait_for=<Future pending cb=[Task.task_wakeup()]> cb=[gather.<locals>._done_callback() at asyncio\tasks.py:720]>```
